Library/ViewModels/DiscoveryPageViewModel.swift

```diff
@@ -241,7 +243,7 @@ public final class DiscoveryPageViewModel: DiscoveryPageViewModelType, Discovery
         }
       }
 
-    self.contentInset = self.viewWillAppearProperty.signal
+    self.contentInset = self.viewWillAppearIsEditorialProperty.signal.filter(isFalse)
```

Comment:
Ahhh sorry about that! 🙈  I actually noticed this and thought it always went under the header haha.

The problem is when the `native_project_cards` experiment is turned on, the the background color needs to be updated so there's enough contrast with the cards :\ 

![Simulator Screen Shot - iPhone 8 - 2020-06-12 at 10 08 32](https://user-images.githubusercontent.com/3156796/84511657-db251800-ac94-11ea-9c2e-91a839d9db14.png)

It's not terrible, I guess.

Another approach would be to prevent the new project cards from displaying if the cards are within the context of an `EditorialViewController`.

Reply:
I'll go with the latter for now.